maestro-client/src/main/java/maestro/Maestro.kt

```diff
@@ -288,6 +288,12 @@ class Maestro(private val driver: Driver) : AutoCloseable {
         driver.close()
     }
 
+    fun takeScreenshot(outFile: File) {
```

Comment:
Would rather have an `OutputStream` or Okio `Sink` here, there might be use cases where we would operate on images in memory